exercises/practice/anagram/.meta/tests.toml

```diff
@@ -45,6 +45,11 @@ description = "detects anagrams using case-insensitive possible matches"
 
 [7cc195ad-e3c7-44ee-9fd2-d3c344806a2c]
 description = "does not detect an anagram if the original word is repeated"
+include = false
+
+[630abb71-a94e-4715-8395-179ec1df9f91]
```

Comment:
Is there a change to the test suite for this?

Reply:
Fixed with the last commit. `s/go Go GO/goGoGO/`